src/main/webapp/WEB-INF/views/login.jsp

```diff
@@ -0,0 +1,16 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Login</title>
+</head>
+<body>
+<h1>Login page</h1>
+<form method="post" action="${pageContext.request.contextPath}/login">
+    Please enter your login: <input type="text" name="login" rewuired>
+    Please enter your password: <input type="password" name="password" required>
+    <button type="submit">Login</button>
+</form>
+<h4><a href="${pageContext.request.contextPath}/drivers/add">Register</a></h4>
+<h4 style="color: red">${errorMessage}</h4>
```

Comment:
add tabs in your jps files.. see default(that were there from the start) jps files for reference